docs/services/activejob.md

```diff
@@ -0,0 +1,7 @@
+# ActiveJob
+
+For some kind of actions to work (for instance, user registration) you need to configure an [ActiveJob](http://edgeguides.rubyonrails.org/active_job_basics.html) backend on your application.
```

Comment:
I think we could clarify this too, "sending emails" would probably be more accurate than "user registration". We could also say that this is required in `production` only (I think `active_job` works syncronously in development by default, and we have letter opener there). What do you think?

Reply:
Ok, how about:  

>  For some kind of actions to work on production (for instance, sending emails on user registration) you need to configure an [ActiveJob](http://edgeguides.rubyonrails.org/active_job_basics.html) backend on your application. 

?

Reply:
Yeah, that works!